test/it/IntegrationSpec.scala

```diff
@@ -70,7 +69,7 @@ abstract class IntegrationSpec
     route(app, request).get
   }
 
-  protected def theCacheFor(pid: String): Option[Answers] = await(cacheRepository.find(Json.obj("providerId" -> "pid")).one[Cache]).map(_.answers)
+  protected def theCacheFor(pid: String): Option[Answers] = await(cacheRepository.find(Json.obj("providerId" -> pid)).one[Cache]).map(_.answers)
```

Comment:
I think this was better as before. Now the `pid` argument isnt used.

Yes it involved us always writing `theCacheFor("pid")` but I like that because it makes it obvious when you glance at the test

Reply:
Sorry ignore me, you fixed it